packages/miniflare/src/index.ts

```diff
@@ -1281,6 +1288,18 @@ export class Miniflare {
 				if (!colors$.enabled) message = stripAnsi(message);
 				this.#log.logWithLevel(logLevel, message);
 				response = new Response(null, { status: 204 });
+			} else if (url.pathname === "/browser/launch") {
+				// Version should be kept in sync with the supported version at https://github.com/cloudflare/puppeteer?tab=readme-ov-file#workers-version-of-puppeteer-core
+				const puppeteer = await npxImport(
+					"puppeteer@22.8.2",
+					this.#log.warn.bind(this.#log)
+				);
```

Comment:
Can I ask why we are dynamically installing puppeteer? I guess it a large library?

I remember we used `npx-import` in the past when d1 was new, and it was quite flakey.

Reply:
@penalosa ?